.travis.yml

```diff
@@ -57,10 +57,16 @@ jobs:
         - docker-compose -f docker-compose.yml -f docker-compose.private-solver.yml up -d stablex
         - cargo test -p e2e ganache -- --nocapture
         - docker-compose logs
+        # StableX e2e Tests (Ganache) - private best-ring solver
```

Comment:
Just to play devils advocate, why do we need to e2e test this solver flavor (we didn't test e.g. the fallback solver)?

Reply:
To catch errors between the interface of the driver and the optimizer. While the fallback solver was mostly a restriction of data (and - at the beginning - different price consumption), this is a substantially different codebase with new preprocessing.